lib/fasttime/fasttime_timing_test.go

```diff
@@ -28,5 +28,16 @@ func BenchmarkTimeNowUnix(b *testing.B) {
 	})
 }
 
+func BenchmarkUnixTime(b *testing.B) {
+	b.ReportAllocs()
+	b.RunParallel(func(pb *testing.PB) {
+		var ts uint64
+		for pb.Next() {
+			ts += uint64(UnixTime().Unix())
+		}
+		atomic.StoreUint64(&Sink, ts)
+	})
+}
```

Comment:
As fast as fasttime.UnixTimestamp().

Reply:
I'm unsure the `UnixTime()` function is needed, since it doesn't improve code readability and maintainability in the place where it is used now. So I reverted this place to easier to read and maintain code, which uses `UnixTimestamp()`.

Reply:
Removed `UnixTime()` function in a follow-up commit